concepts/Common/Integration Guides/50 Use MUI components in your DevExtreme application/50 Use MUI components in your DevExtreme application.md

```diff
@@ -0,0 +1,191 @@
+        text: {
+        primary: '#173A5E',
+        secondary: '#EBE7E7',
+        },
+        action: {
+        active: '#001E3C',
+        },
+        success: {
+        dark: '#009688',
+        },
+        error: {
+        main: '#FF6666'
+        }
+    },
+    });
+
+Enclose the content of the page in the `ThemeProvider` tag to apply the theme:
+
+    <!--tab: App.tsx-->
+    <ThemeProvider theme={theme}>
+    <MUIFormDxEditors />
+    ...
+    </ThemeProvider>
+
+### Styling Non-MUI components
+
+Non-MUI components do not automatically gain access to the `sx` styling prop. Use the `unstable_styleFunctionSx` utility to create a wrapper for the DxButton component:
+
+    <!--tab: MUI-Form-with-DX-editors.tsx-->
+    import { useTheme } from 'styled-components';
+    import { unstable_styleFunctionSx, SxProps, styled, Theme }from '@mui/system';
+    interface DivProps {
+        sx?: SxProps;
+    }
+    const DxButtonSxWrapped = styled(DxButton)<DivProps>( unstable_styleFunctionSx,);
+
+Now we can retrieve the global MUI theme and style the button: 
+
+    <!--tab: MUI-Form-with-DX-editors.tsx-->
+    const theme = useTheme() as Theme;
+    <DxButtonSxWrapped sx={{
+        'backgroundColor': theme.palette.error.main,
+        'color': theme.palette.text.secondary
+    }} text="Reset Form" onClick={handleReset} 
+    />
+
+This approach is useful if your MUI theme includes settings that DevExtreme themes cannot store --- for example, font-related settings. The easier way to style DevExtreme components nested within MUI components is to simply use DevExtreme themes. Open the [DevExtreme theme builder](https://devexpress.github.io/ThemeBuilder/), define the same color palette as the MUI theme, and reference the DevExtreme theme in your application.
+
+## MUI components inside DevExtreme components
+
+To use MUI controls inside a DevExtreme components, we need to apply a similar strategy.
+
+Create functions that handle form state change:
+
+    <!--tab: DX-Form-with-MUI-editors.tsx>
+    const [formData, updateFormData] = useState<FormData>({
+        textBox: '',
+        muiInput: '',
+        checkBox: false
+    });
+    const handleFormDataChange = (name: string) => (value: string | boolean | null) => {
+        updateFormData({ ...formData, [name]: value });
+    }
+    const handleMuiFormDataChange = (name: string) => (event: React.ChangeEvent<HTMLInputElement>) => {
+        updateFormData({ ...formData, [name]: event.target.value });
+    }
+
+Create a wrapper for MUI inputs that triggers these functions:
+
+    <!--tab: DX-Form-with-MUI-editors.tsx>
+    const InputTemplate = () => {
+        return <MUIInput onChange={handleMuiFormDataChange('muiInput')} />;
+    }
+
+Wrap form elements inside `DXItem` and `DXLabel` components. Call the [render()](https://js.devexpress.com/React/Documentation/Guide/React_Components/Component_Configuration_Syntax/#Markup_Customization/Using_a_Rendering_Function) method to render MUI components.
```

Comment:
I'd like to ask about the flow of these two sentences. How does the second sentence relate to the first?